bigchaindb/backend/schema.py

```diff
@@ -16,10 +16,14 @@
 
 import bigchaindb
 from bigchaindb.backend.connection import connect
+from bigchaindb.common.exceptions import ValidationError
 
 logger = logging.getLogger(__name__)
 
 TABLES = ('bigchain', 'backlog', 'votes', 'assets')
+VALID_LANGUAGES = ('danish' 'dutch' 'english' 'finnish' 'french' 'german'
+                   'hungarian' 'italian' 'norwegian' 'portuguese' 'romanian'
+                   'russian' 'spanish' 'swedish' 'turkish')
```

Comment:
Why are there no commas between these strings, similar to `TABLES`?

Also, according to https://docs.mongodb.com/manual/reference/text-search-languages/ , MongoDB text search also supports some two- and three-letter language codes such as `en`, `ara` and `zhs`.